drivers/sensor/mhz19b/mhz19b-priv.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __MHZ19B_PRIV_H__
```

Comment:
Shouldn't you use `ZEPHYR_DRIVERS_SENSOR_MHZ19_MHZ19B_PRIV`?

Reply:
Yes you are right, fixed it, thanks!